src/Microsoft.TestPlatform.CommunicationUtilities/SocketConstants.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.CommunicationUtilities
+{
+    public class SocketConstants
+    {
+        // Buffer size for the buffered stream we are using.
+        public const int BUFFERSIZE = 16384;
```

Comment:
Nit: Isn't the MS standard to use PascalCase for constant names?

Reply:
Yeah, Thanks !

---
In reply to: [137451610](https://github.com/Microsoft/vstest/pull/1041#discussion_r137451610) [](ancestors = 137451610)